opm/autodiff/StandardWell.hpp

```diff
@@ -149,10 +139,15 @@ namespace Opm
         /// computing the well potentials for group control
         virtual void computeWellPotentials(const Simulator& ebosSimulator,
                                            const WellState& well_state,
-                                           std::vector<double>& well_potentials) const;
+                                           std::vector<double>& well_potentials) /* const */;
```

Comment:
Why is this no longer a const function?

Reply:
Ignore my question, I see why now. We can consider the const method/mutable member balance later.